src/server/pkg/deploy/cmds/cmds.go

```diff
@@ -255,7 +255,8 @@ func DeployCmd(noMetrics *bool, noPortForwarding *bool) *cobra.Command {
 				}
 				cred = string(credBytes)
 			}
-			if err = assets.WriteGoogleAssets(manifest, opts, args[0], cred, volumeSize); err != nil {
+			bucket := strings.TrimPrefix(args[0], "gs://")
+			if err = assets.WriteGoogleAssets(manifest, opts, bucket, cred, volumeSize); err != nil {
```

Comment:
This is a nice ergonomic improvement. Should we do a similar thing for `s3://`, `wasb://` for those object stores.

Reply:
We actually do this for `s3://` already, and I just added `wasb://`